app/graphql/mutations/create_supporter.rb

```diff
@@ -0,0 +1,27 @@
+class Mutations::CreateSupporter < Mutations::BaseMutation
+  argument :name, String, required: true
+  argument :email, String, required: true
+  argument :need, ID, required: true
+
+  field :supporter, Types::SupporterType, null: true
+  field :errors, [String], null: false
+
+  def resolve(name:, email:, need:)
+    supporter = Supporter.new(
+              name: name,
```

Comment:
I like that we're collecting the supporters name so that in the future we can have admin functionality (an admin to reach out to a volunteer).
And love the ability to connect the need to the volunteer now.